nano/node/rpc.cpp

```diff
@@ -3486,8 +3486,36 @@ void nano::rpc_handler::wallet_representative_set ()
 		nano::account representative;
 		if (!representative.decode_account (representative_text))
 		{
-			auto transaction (node.store.tx_begin_write ());
-			wallet->store.representative_set (transaction, representative);
+			{
+				auto transaction (node.store.tx_begin_write ());
+				wallet->store.representative_set (transaction, representative);
+			}
+			// Change representative for all wallet accounts
+			if (request.get<bool> ("force", false))
```

Comment:
Since current the semantics of `wallet_representative_set` is to set representative for new accounts only, what do you think about calling this parameter `update_existing_accounts` ?

Reply:
makes sense